### PR TITLE
Custom labels for all pods

### PR DESCRIPTION
**Pull request by orrshilon**
* Add `labels` value which is appended to default labels in all pods in the chart

### CODE REVIEW

File: charts/rasa-x/values.yaml [hunk 1]
```diff
@@ -99,6 +99,10 @@ rasax:
     # annotations for the service
     annotations: {}
 
+  # additional labels can be used to map organizational structures onto system objects
+  # https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+  labels: {}
```
Comment (tczekajlo) [inline charts/rasa-x/values.yaml:104]: I'd move these fields to the `global` section and change the name to `additionalDeploymentLabels`. The `labels` field in the `rasax` section implies that extra labels are applied only for the rasa-x deployment.

File: charts/rasa-x/values.yaml [hunk 2]
```diff
@@ -99,6 +99,10 @@ rasax:
     # annotations for the service
     annotations: {}
 
+  # additional labels can be used to map organizational structures onto system objects
```
Comment (tczekajlo) [inline charts/rasa-x/values.yaml:102]: The comment should start with the name of the property that it describes, more: https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml

**Review by tczekajlo — CHANGES_REQUESTED** [review]
@orrshilon Thanks a lot for your first PR in the project 🎉  I've left two small comments 😄

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @akelad will take a look at it as soon as possible ✨

**Review by tczekajlo — APPROVED** [review]
LGTM

**Pull request merged**